content-review/impl/turnitin/src/main/java/org/sakaiproject/contentreview/turnitin/TurnitinReviewServiceImpl.java

```diff
@@ -371,7 +371,7 @@ public String getReviewReportInstructor(String contentId, String assignmentRef,
 		Map params = TurnitinAPIUtil.packMap(turnitinConn.getBaseTIIOptions(), "fid", fid, "fcmd", fcmd, "assignid",
 				assignid, "cid", cid, "oid", oid, "utp", utp);
 
-		params.putAll(getInstructorInfo(item.getSiteId()));
+		params.putAll(getInstructorInfo(item.getSiteId(), false, userId));
```

Comment:
This is the big change ..... old call would have returned the first instructor in the site ......

Untested .....